client/src/components/JobSections/JobDetails/index.tsx

```diff
@@ -0,0 +1,212 @@
+import { Button, Grid, InputAdornment, TextField, ToggleButton,
+     ToggleButtonGroup, 
+     Typography} from "@mui/material";
+import { Save } from '@mui/icons-material';
+import { ColorPicker, Color } from 'material-ui-color';
+import { useState, useEffect, SetStateAction, Dispatch } from "react";
+import { ApplicationObject } from "typings";
+
+export default function JobDetails(props:
```

Comment:
define a PropType interface above here and use it so it does 
`props: PropType`